src/components/UserSelector.tsx

```diff
@@ -68,12 +57,17 @@ export const UserSelector: React.FC<Props> = ({
 
       <div className="dropdown-menu" id="dropdown-menu" role="menu">
         <div className="dropdown-content">
+          {users.length === 0 && (
+            <div className="dropdown-item is-disabled">No users available</div>
+          )}
           {users.map(user => (
             <a
               key={user.id}
               href={`#user-${user.id}`}
-              onClick={() => {
+              onClick={e => {
+                e.preventDefault();
                 onChange(user);
```

Comment:
Ensure that the `onChange` function correctly updates the selected user state in the parent component.